alpha/serendipity_event_challengeresponse.php

```diff
@@ -16,7 +16,7 @@ function introspect(&$propbag) {
             'smarty'      => '2.6.7',
             'php'         => '4.1.0'
         ));
-        $propbag->add('version',       '0.10');
+        $propbag->add('version',       '0.11');

```

Comment:
Not important enough to delay anything, just for the feature: 0.10.1 to notice it is just a bugfix release would be better. We are not consistent with it, but try to use semver where it fits.